app/js/lib/foxbox/services/base.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+const TYPE = 'Unknown';
```

Comment:
__optional nit:__ If it's not user facing string, maybe it will be more convenient to use lower-case only strings (```unknown, ip-camera, etc```)? Maybe it's not a big deal in the end, but sometimes it costs developer time to notice case-related typo :)